x-pack/plugins/index_management/public/application/sections/home/template_list/template_table/template_table.tsx

```diff
@@ -89,15 +108,30 @@ export const TemplateTable: React.FunctionComponent<Props> = ({
       sortable: true,
     },
     {
-      name: i18n.translate('xpack.idxMgmt.templateList.table.overridesColumnTitle', {
-        defaultMessage: 'Overrides',
+      name: i18n.translate('xpack.idxMgmt.templateList.table.dataStreamColumnTitle', {
+        defaultMessage: 'Data stream',
+      }),
+      truncateText: true,
+      render: (template: TemplateListItem) =>
+        template._kbnMeta.hasDatastream ? <EuiIcon type="check" /> : null,
+    },
+    {
+      name: i18n.translate('xpack.idxMgmt.templateList.table.contentColumnTitle', {
+        defaultMessage: 'Content',
       }),
       truncateText: true,
       render: (item: TemplateListItem) => (
         <TemplateContentIndicator
           mappings={item.hasMappings}
           settings={item.hasSettings}
           aliases={item.hasAliases}
+          contentWhenEmpty={
```

Comment:
Just a note:

I'm pretty familiar with index templates and it took me a while to figure out what `M S A` meant in this context. I'd love to see this spelled out in some way, but even a tooltip might help.

Part of me wonders if this information needs to be surfaced at this level at all.

Not necessarily actionable or blocking. Just thoughts to consider.

Reply:
Didn't the tooltip show up on your side? There is one for each letter

<img width="163" alt="Screenshot 2020-07-08 at 10 33 56" src="https://user-images.githubusercontent.com/2854616/86896674-9dd26f80-c106-11ea-8464-ae0157774034.png">